receiver/windowseventlogreceiver/receiver_windows_test.go

```diff
@@ -301,6 +302,10 @@ func createTestConfig() *WindowsLogConfig {
 // It returns a function that can be used to uninstall the event source, that function is never nil
 func assertEventSourceInstallation(t *testing.T, src string) (uninstallEventSource func(), err error) {
 	err = eventlog.InstallAsEventCreate(src, eventlog.Info|eventlog.Warning|eventlog.Error)
+	if err != nil && strings.HasSuffix(err.Error(), " registry key already exists") {
```

Comment:
Checking for the string because `eventlog.InstallAsEventCreate` doesn't return the win32 API error, but, a generic error with an english string, see https://cs.opensource.google/go/x/sys/+/refs/tags/v0.25.0:windows/svc/eventlog/install.go;l=44